File: Python/8py 1/difference_in_ages.py
"""
|--------------------------------------------------------------------------
| Find the Difference in Age between Oldest and Youngest Family Members
|--------------------------------------------------------------------------
|
"""

def difference_in_ages(ages):
   return (min(ages), max(ages), max(ages)-min(ages))


def difference_in_ages(ages):
   x, y = min(ages), max(ages)
   return x, y, y-x


def difference_in_ages(ages):
   age = sorted(ages)
   return (age[0], age[-1], (age[-1]-age[0]))


print(difference_in_ages([0, 110]), (0, 110, 110))
print(difference_in_ages([33, 33, 33]), (33, 33, 0))

File: PHP/8php 1/differenceInAges.php
<?php
/*
|--------------------------------------------------------------------------
| Find the Difference in Age between Oldest and Youngest Family Members
|--------------------------------------------------------------------------
|
*/

function differenceInAges($ages) {
   return [min($ages), max($ages), max($ages)-min($ages)];
}


function differenceInAges($a){
   return array(min($a) ,max($a), max($a)-min($a));
}


function differenceInAges($ages) {
   return $arr = ["0" => min($ages),
                  "1" => max($ages),
                  "2" => max($ages)-min($ages)
   ];
}


print_r(differenceInAges([82, 15, 6, 38, 35]));
print_r(differenceInAges([57, 99, 14, 32]));